step-core/src/main/java/step/core/dynamicbeans/DynamicBeanResolver.java

```diff
@@ -92,6 +93,21 @@ public void evaluate(Object o, Map<String, Object> bindings) {
 		}
 	}
 
+	public void evaluateMandatoryFieldsEvenForSkippedArtefact(AbstractArtefact artefact, Map<String, Object> bindings) {

```

Comment:
The DynamicBeanResolver is not specific to AbstractArtefacts and applies to objects in general. For this reason I wouldn't add this specific method here. It would be nicer to make evaluateDynamicValue public and declare this method in a more specific class (like ArtefactHandler for instance=